src/plugin/applet.c

```diff
@@ -231,7 +231,7 @@ GSettings *budgie_applet_get_applet_settings(BudgieApplet *self, gchar *uuid)
         GSettings *settings = NULL;
         gchar *path = NULL;
 
-        if (!self->priv->schema || !self->priv->prefix) {
+        if (!self || !self->priv || !self->priv->schema || !self->priv->prefix) {
```

Comment:
Looks like a bigger bug is afoot imo - we shouldn't be able to call get_applet_settings unless we're already dealing with a NULL pointer somewhere